TheFourFifths/consus-client
# [TFF-153] Longterm - New checkout

### Summary

You can make a new checkout become long-term by clicking the long-term checkout checkbox before completing checkout. I still need tests and to add documentation

### Checklist


### How to Review

1. Start server on new-checkout-tff153
2. Get an itemAddress(iGwEZUvfA)
3. Lookup student 123456
4. Add item to cart
5. Click Long-term checkout checkbox
7. Try to checkout verify error msg about missing due date
8. Enter due date
9. Try to checkout verify error msg about missing prof
10. Enter professor
11. Complete checkout
12. It's a long term checkout I promise?

### Links

- [TFF-29](https://msoese.atlassian.net/browse/TFF-153)
- TheFourFifths/consus/pull/59
